pkg/utils/utils.go

```diff
@@ -70,6 +86,26 @@ func GetValidCniPath(inGoPath string) (string, error) {
 	return cniPath, nil
 }
 
+func GetValidKubeConfig() string {
+	kcPath := filepath.Join(pwd, kcRelPath)
```

Comment:
For some reason this doesn't work for me. It prints:

```
2017/08/08 11:54:37 Note: For kubectl to work, please set $KUBECONFIG to .kube-spawn/default/kubeconfig
```

which is not good because setting `$KUBECONFIG` to that will only make kubectl work if I'm in the kube-spawn directory.

Can you use `os.Getwd()` instead?